.gitignore

```diff
@@ -41,3 +41,8 @@ yarn-error.log*
 out/
 !**/src/main/**/out/
 !**/src/test/**/out/
+node_modules/
```

Comment:
There is already `/node_modules` in .gitignore - is there need for this or can it be removed?